### DISCUSSION TITLE
Copilot Billing

### DISCUSSION THREAD

**Discussion by shubhamshrey29** [community/community#124043] [Copilot Conversations]
### Select Topic Area

General

### Body

Hey, can anyone tell me why I am billed twice for copilot? on the 10th I bought a Copilot subscription, a mail followed it that my copilot free trial was active, but I already paid 10$. 

Following that I decided to cancel the subscription on the 14th as I don't want to pay 10$ next month. So they charged 8$ for the remaining 27 days. I asked them why they canceled my subscription and only refunded 8$. They provided with this answer.

Hi there,
 
Thanks for writing back!
 
It appears that on 2024-05-10 that was an authorization hold that was done by the payment processor when the new card was added to the account, not an actual charge. Typically the amount of the authorization hold is released back to the account rather quickly, but depending on your bank it may be unavailable for a longer period.
 
As an authorization is not a charge, I'm afraid we don't have any money to refund on our end.
 
I would recommend reaching out to your credit card provider directly for more information about the transaction and when you might expect to see the returned funds back in the account. I can confirm that no payment has been processed by GitHub on 2024-05-10, no money was taken.
 
 
Please let us know if you have any questions, or if we can help with anything else at the moment.
 
Best,
GitHub Support

will I get my money back. can anyone provide me solution.

**Comment by dec1costello** [comment]
I'm curious if github gives out free copilot subscriptions to amend this, was that ever offered?

**Reply by shubhamshrey29** [reply]
Not offered anything they just stopped replying to the ticket.
